### PR TITLE
#333 - cookies component

### LINKED ISSUE #333: Cookies component

### PR DESCRIPTION
**Pull request by AleksandraKozubal**
Added cookies component.
image
image

It should close #333.

### CODE REVIEW

**Comment by EwelinaSkrzypacz** [comment]
Conflicts :(

**Review by EwelinaSkrzypacz — DISMISSED** [review]

File: resources/views/components/cookies.blade.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    this.dispatchEvent()
+                }
+            },
+
+            dispatchEvent() {
+                document.dispatchEvent(new CustomEvent('cookieConsent', {
+                    detail: this.state
+                }))
+            }
+
```
Comment (kamilpiech97) [inline resources/views/components/cookies.blade.php:29]:
```suggestion
            init() {
                this.dispatchEvent()
            },
            dialogue: {
                ['x-show']() {
                    return this.state === 'unknown'
                }
            },
            accept: {
                ['@click']() {
                    this.state = 'accepted'

                    this.dispatchEvent()
                }
            },
            dispatchEvent() {
                document.dispatchEvent(new CustomEvent('cookieConsent', {
                    detail: this.state
                }))
            }
```

Comment (AleksandraKozubal) [reply resources/views/components/cookies.blade.php:29]: What's the difference?

Comment (kamilpiech97) [reply resources/views/components/cookies.blade.php:29]: no enters :eyes:

**Review by kamilpiech97 — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**